src/modules/sbstudio/plugin/operators/create_takeoff_grid.py

```diff
@@ -262,8 +234,17 @@ class CreateTakeoffGridOperator(Operator):
         unit="LENGTH",
     )
 
-    intra_slot_spacing = FloatProperty(
-        name="Intra-slot spacing",
+    intra_slot_spacing_row = FloatProperty(
+        name="Intra-slot spacing row",
+        description="Spacing between drones in the same slot of the grid",
+        default=0.5,
+        soft_min=0,
+        soft_max=5,
+        unit="LENGTH",
+    )
+
+    intra_slot_spacing_col = FloatProperty(
+        name="Intra-slot spacing column",
         description="Spacing between drones in the same slot of the grid",
```

Comment:
Description needs to be updated to reflect that this is now for a single column only.